docs/get-started/video.md

```diff
@@ -48,7 +48,7 @@ Create and activate a virtual environment, then choose an install option:
 
 ::::{tab-set}
 
-:::{tab-item} With internvideo2
+:::{tab-item} PyPi With internvideo2
```

Comment:
Please reorder this tab so that we have:
PyPi Without internvideo2 -> Source without internvideo2 -> PyPi with internvideo2 -> Source with internvideo2 -> nemo curator Container (IMO this is the preferred order)